src/main/webapp/dev/js/main/instructorCourseEdit.es6

```diff
@@ -473,6 +473,20 @@ $(document).ready(() => {
         showNewInstructorForm();
     });
 
+    $("[name='instructorisdisplayed']").change((e) => {
```

Comment:
Use single quotes at the start and end of a string, and use double quotes within the string. Note that this rule applies to JS only.